README.md

```diff
@@ -47,6 +47,23 @@ apply(plugin = "com.deliveryhero.whetstone")
 
 This automatically configures Dagger and Anvil, and also adds the necessary whetstone dependencies for you.
 
+Finally, in your :app module. You need to enable the kapt plugin, anvil requires this in the app module so it can generate the necessary components.
```

Comment:
An idea here might be to also do these configurations transparently with the whetstone-gradle-plugin, since we already mention that whetstone sets these things up for you anyway

Basically, when a user applies the plugin:

for app module:
- install anvil (done)
- disable factories generation (done)
- install kapt for dagger compiler **(todo)**

for lib module:
- install anvil (done)
- enable factories generation (done)
- warn if kapt for dagger compiler is found **(todo)**

What do you think?

Reply:
yup. that's a plan. will get to it.